app/models/manageiq/providers/embedded_ansible/automation_manager/configuration_script.rb

```diff
@@ -2,4 +2,8 @@ class ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ConfigurationScri
   ManageIQ::Providers::EmbeddedAutomationManager::ConfigurationScript
 
   include ManageIQ::Providers::AnsibleTower::Shared::AutomationManager::ConfigurationScript
+
+  def jobs
+    ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Job.where(:orchestration_template_id => id)
```

Comment:
👍 - Looks good to me - With the caveat of eventually needing a follow up PR to handle the External provider.